src/electron/build_common.action.sh

```diff
@@ -43,3 +43,5 @@ webpack \
 # TODO: make non-packaged builds work without this
 node scripts/environment_json.mjs "${PLATFORM}" \
     --buildMode="${BUILD_MODE}" > www/environment.json
+
+electron-icon-maker --input=resources/electron/icon.png --output=build
```

Comment:
so... this was the only difference between `build` and `package_common` - i didn't see the utility in having two different scripts, so i've merged them. lmk if there's something i'm ignorant of

Reply:
The semantics are confusing. Let's have a single build action (revert the name) that builds everything we need before electron-builder.

Move the package_windows logic to generate the env file here (and remove the existing duplication). It will have to be in an if-block based on the platform.

Once you call this script, all you need is to call electron-builder, which seems to take different options based on the mode.

Reply:
Yep, that's basically what I was thinking. Exciting!